sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcUtil.java

```diff
@@ -443,7 +442,7 @@ private static Calendar withTimestampAndTimezone(DateTime dateTime) {
    *
    * @param <PartitionT>
    */
-  interface JdbcReadWithPartitionsHelper<PartitionT>
+  public interface JdbcReadWithPartitionsHelper<PartitionT>
```

Comment:
JdbcUtil is still package private. One would not be able to instantiate JdbcReadWithPartitionsHelper outside the same package.

Reply:
Fixed